dev-util/gn/patches/gn-091169b.patchset

```diff
@@ -0,0 +1,404 @@
+From 2ce32f78997886a3875f7dbf3e3ea0547fe31d50 Mon Sep 17 00:00:00 2001
+From: Calvin Hill <calvin@hakobaito.co.uk>
+Date: Wed, 26 Sep 2018 20:48:23 +0100
+Subject: gn: Haiku support
+
+
+diff --git a/base/files/file.h b/base/files/file.h
+index 61239ad..bea5094 100644
+--- a/base/files/file.h
++++ b/base/files/file.h
+@@ -22,10 +22,10 @@
+ 
+ namespace base {
+ 
+-#if defined(OS_BSD) || defined(OS_MACOSX) || defined(OS_NACL) || \
++#if defined(OS_BSD) || defined(OS_MACOSX) || defined(OS_NACL) || defined(OS_HAIKU) || \
```

Comment:
Well this change is needed since it defines `stat` in [line 18.](https://github.com/haikuports/haikuports/blob/ac07292338db2a9b07fbea7eb03c2b7ebd04b567/dev-util/gn/patches/gn-091169b.patchset#L18) removing that causes the build to fail.